### ISSUE TITLE
How ti

### ISSUE THREAD

**Issue by xiang-jian-wen** [huawei-noah/noah-research#43]
Hello sir:
I downloaded the oxfordiiitpet data set from the official website and conducted experiments according to the published training set and test set, but the results did not achieve good results. May I ask what is the reason?Is it the training strategy?

**Comment by xiang-jian-wen** [comment]
Hello sir:
I downloaded the oxfordiiitpet data set from the official website and conducted experiments according to the published training set and test set, but the results did not achieve good results. May I ask what is the reason?Is it the training strategy?
